src/js/view/controls/tizen/tizen-controlbar.ts

```diff
@@ -277,6 +272,10 @@ export default class TizenControlbar extends Controlbar {
         }
     }
 
+    toggleVisibility(visible: boolean): void {
+        toggleClass(this.el.querySelector('.jw-bottom-container'), 'jw-hidden', !visible);
```

Comment:
Did you just want to toggle visibility of the `bottom-container`? This is the timeslider and play button, but the back button and settings button would still be visible since they are in the `top-container`. If you wanted to hide the whole controlbar, that could be `jw-tizen-controlbar`.

Reply:
Yes. The controlbar is the only thing I want to toggle, but the class is written in a way where it covers both the upper buttons and the bar.